### ISSUE TITLE
Changes continously breaks other parts of the app

### ISSUE THREAD

**Issue by nspartners** [stackblitz/bolt.new#1456]
Everytime a fix is made in one area of the project, it breaks another area and it goes on in an endless loop of break + fix.  Many hours and 10M tokens later, I'm stuck with any new change breaking another part of the application.

**Comment by Frankg40** [comment]
Yep.

The AI loses context of your entire project and doesnt take into consideration the tentacles of each function that reach into other parts of the app that may be affected. It doesn't remember what it did previously and often forgets to restore code that didnt need deleted when adding new code to a file for a new function.

There is an inherit flaw in how it's built. With that said Im pretty sure they know that and are working on exactly that problem.

**Comment by ranaprathap2** [comment]
Yes facing the same issue it really troublesome.  Its really a showstopper that prevents me from using bolt when scaling my app. Hope they address and fix this soon ASAP.

**Comment by yasinkocak** [comment]
Facing the same issue, this should be a high priority bug
